styles/scss/components/menu/menu-overlay.scss

```diff
@@ -0,0 +1,65 @@
+.video-react-options-close {
+  position: absolute;
+  top: 0.4em;
+  right: 0.5em;
+  color: white;
+  font-size: 2.5em;
+  z-index: 2;
+
+  &:hover {
+    cursor: pointer;
+    opacity: 0.8;
+  }
+}
+
+.video-react-options-overlay {
+  position: absolute;
+  top: 0;
+  right: 0;
+  height: 100%;
+  width: 325px;
+  background: rgba(#000000, 0.8);
+  z-index: 1;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  font-family: "Circular", sans-serif;
+
+  span {
+    font-size: 1.5em;
+  }
+
+  .video-react-options-menu-section {
+    width: 80%;
```

Comment:
Non-blocking feedback. I know we talked about this in person too, just documenting. Not super important at the moment / for this one case, but I feel percentage might lead us astray in situations like these. I expect that the empty space we achieve from this percentage would be better captured as an absolute value in `em`s, `rem`s, or `px`s. And likely it would be best applied as padding to the container of `.video-react-options-menu-section`, ensuring that all children keep a certain distance from the border (which, out of context, sounds a little dystopian).